cdc/log.cc

```diff
@@ -1150,7 +1150,7 @@ struct process_row_visitor {
                     }
                 };
 
-                std::vector<bytes> buf;
+                std::list<bytes> buf;
```

Comment:
It seems that buf is never used outside udt_visitor here so maybe we could just encapsulate it inside udt_visitor class? In a separate commit of course.

Reply:
This will probably go away soon anyway.